core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalWriter.java

```diff
@@ -116,6 +127,7 @@ public void flush() throws IOException, JournalClosedException {
             mWriteTimeoutMs), e);
       }
       mJournalEntryBuilder = null;
+      mCurrentJournalEntrySize.set(0);
```

Comment:
(nit) is it necessary since you have set it in line 87?

Reply:
No necessary, removed! Good catch